utils/getStates.js

```diff
@@ -49,6 +50,21 @@ module.exports = async (
 		if (!json) {
 			spinner.info(`${cyan(`Sorted by:`)} ${sortBy}${isRev}`);
 		}
+
```

Comment:
`corona states` throws an error when the output folder does not exist. Should create the directory if it doesn't exist before writing the records.